fix missing StringArray.h

Currently the actions [fail](https://github.com/frankalicious/yoradio/actions/runs/16527033151/job/46742906705).  
  
```
src/core/../displays/widgets/pages.h:5:10: fatal error: StringArray.h: No such file or directory
```

--- a/yoRadio/platformio.ini
+++ b/yoRadio/platformio.ini
@@ -26,7 +26,7 @@ lib_deps =
   adafruit/Adafruit BusIO@^1.14.5
   adafruit/Adafruit GFX Library@^1.11.9
   bblanchon/ArduinoJson@^7.0.4
-  https://github.com/me-no-dev/ESPAsyncWebServer
+  https://github.com/me-no-dev/ESPAsyncWebServer#b70d36f
 
 platform_packages =
 	platformio/framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git#3.0.4

